docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -129,54 +132,22 @@ works in ev3dev - [gattlib](https://bitbucket.org/OscarAcena/pygattlib)
 
     sudo pip install gattlib
 
-This library is also used as an extension for a more known library, [pybluez](https://pypi.python.org/pypi/PyBluez)
-so if you want a library for both bluetooth "Classic" and BLE this would be better:
-
-    pip install pybluez
-    pip install pybluez[ble]
-
-Unfortunately I couldn't make it work in my ev3dev system.
-
-Please note that it takes **a lot** of memory and around 2 hours to install gattlib.
-After some failures ("virtual memory exhausted: Cannot allocate memory") I finally
-succeeded extending my ev3dev swapfile to almost 1 GB (please first check if you have
-enough free space in your SD card):
-
-    robot@ev3dev:~$ sudo dd if=/dev/zero of=/swapfile1 bs=1024 count=917504
-    917504+0 records in
-    917504+0 records out
-    939524096 bytes (940 MB) copied, 442.332 s, 2.1 MB/s
-
-it will take 5 to 10 minutes to allocate space for such a big file
-
-    robot@ev3dev:~$ sudo mkswap /swapfile1
-    Setting up swapspace version 1, size = 917500 KiB
-    no label, UUID=55fcb430-451b-4699-955c-5754bf65999b
-
-    robot@ev3dev:~$ sudo swapon /swapfile1
-    swapon: /swapfile1: insecure permissions 0644, 0600 suggested.
-
-this is a temporary measure so we'll skip security warning
-
-    robot@ev3dev:~$ sudo swapon -s
-    Filename				Type		Size	Used	Priority
-    /dev/zram0                             	partition	98300	8188	16383
-    /swapfile1                             	file    	917500	0	-1
-
-We don't want to use the swapfile in memory (it will overflow) so we disable it:
+Since the beginning of 2017 (snapshot-ev3dev-jessie-ev3-generic-2017-01-16) this
+library is already included in the ev3dev image. You can check the version of the
+installed package with:
 
-    sudo systemctl stop zram_swap.service
+    dpkg -s python3-gattlib | grep "Version:"
+    Version: 0.20150805-1ev3dev1
 
-After installation completes we reset the swapfile configuration:
+If version is older (or inexistent) you should update/install it:
```

Comment:
Yes, I keep forgetting these english grammar quirks... most portuguese words translate easily to english and since we have "existente" and "inexistente" it was automatic...